### DISCUSSION TITLE
Change end of fill bar to be flat?

### DISCUSSION THREAD

**Discussion by coffinmilk** [AlexPerathoner/SlimHUD#31] [Ideas]
Although most might prefer the inside of the bar to be rounded, for iOS consistency, can the bar be made flat? 
![IMG_7046](https://user-images.githubusercontent.com/99998699/189546713-85ae5a15-a31f-4625-b5b9-f6ff4527e536.jpg)
<img width="378" alt="Screenshot 2022-09-11 at 14 59 37" src="https://user-images.githubusercontent.com/99998699/189546715-2e83f0ed-be23-47da-bb5c-281a331c947e.png">

**Comment by AlexPerathoner** [comment]
Added in release [v1.4.1](https://github.com/AlexPerathoner/SlimHUD/releases/tag/v1.4.1)
